Testsuite/test/collect_cgal_testresults_from_cmake

```diff
@@ -53,12 +55,12 @@ print_testresult()
       then
         RESULT="w"
       else
-	if grep -E -q 'NOTICE: .*(need|require|incompatible).*and.*will not be' CompilerOutput_$1
-	then
-	  RESULT="r"
+  if grep -E -q 'NOTICE: .*(need|require|incompatible).*and.*will not be' CompilerOutput_$1
+  then
+    RESULT="r"
         else
           RESULT="y"
-	fi
+  fi
```

Comment:
The indentation is still wrong around here.

Reply:
I base dit on master instead of 5.1.x, the indentation is not the only thing wrong in that diff